pages/vi/github-issues.md

```diff
@@ -65,13 +65,13 @@ Now you can go and make the proposed changes to your local files. You can use an
 | --- |
 If you're still confused, that's quite alright. [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow) can be quite challenging at first. See this diagram below. For better context, the new branch you just made is inside your "Forked GitHub IO".
 
-## NEED DIAGRAM HERE
+![flowchart](/images/flowchart.png)
```

Comment:
image does not show on raw.githack so can you put relative path here?

Reply:
good catch I have put the relative path so now the image shows on raw.githack link